docs/reference/glossary.md

```diff
@@ -0,0 +1,194 @@
+# Glossary
+
+* **Artifacts**
+
+    Any files or resources generated during the software development lifecycle, including source code, binary files, and compiled packages. In Devtron, you can view the artifacts in the `Build History` tab of your application.
+
+* **Base Deployment Template**
+
+    It’s a single entry point (an all-in-one YAML file) for you to enter the values, so that when the application is deployed the filled values go to the respective template files (YAML), and accordingly the resources would be created. In Devtron, you get the option to select a base deployment template in the `App Configuration` tab at the time of creating an application.
+
+* **Build Context**
+
+    The set of files and directories used as input when building a Docker container image using a Dockerfile. Specify the set of files to be built by referring to a specific subdirectory, relative to the root of your repository. To build all files from the root, use (.) as the build context, or set build context by referring a subdirectory path such as /myfolder or /myfolder/buildhere if path not set, default path will be root dir of selected git repository.
```

Comment:
This too is incorrect we don't need a set of files as an input. Let's say for building a docker image: we require a dockerfile and context. Dockerfile contains the instructions to build. Context is the path where the build process may refer for files required for build